pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -49,6 +50,8 @@ const (
 	successCount uint = 1
 	// Timeout when we check whether deferred messages should be retried
 	sendTimeoutInSecs uint32 = 15
+	// Wait for atleast 10 minutes before purging app url metrics
+	appURLMetricsPurgeTimeout = 600
```

Comment:
The current implementation looks fine, but the comment and name here might be misleading.
Every 10 minutes the code purges all of the app instances which no longer exist, but the "timeout" might lead the reader to think the metrics are reported for 10 minutes after the app instance was deleted which is not the case.

Reply:
will fix